index.js

```diff
@@ -108,6 +108,19 @@ module.exports = {
     'no-undef-init': 'error',
     'no-undefined': 'off',
     'no-unused-vars': 'warn',
-    'no-use-before-define': ['error', {functions: false}]
+    'no-use-before-define': ['error', {functions: false}],
```

Comment:
Ah, I remember. Instead of having every single one of these rules defined, we base our linting off of `standard`, and then just override the three rules we care strongly about (and `standard` is supported by Code Climate). You can decide whether of not having the explicit declaration is more beneficial.